static/templates/user_info_popover_content.handlebars

```diff
@@ -32,7 +32,19 @@
         <li>{{ user_time }} {{#tr this}}Local time{{/tr}}</li>
         {{/if}}
 
-        <li>{{ user_type }}</li>
+        {{#if is_bot}}
+            {{#if bot_owner}}
+            <li>{{#tr this}}Owner{{/tr}}:
+                <span class="bot-owner-name" data-bot-owner-id='{{ bot_owner.user_id }}'>
+                    {{bot_owner.full_name}}
+                </span>
+            </li>
+            {{else}}
+            <li>System Bot</li>
```

Comment:
I think this should only be "System Bot" if it's a cross-realm bot; otherwise it should be just "Bot".

(Also, tag this string for translation)

Reply:
If it's not a cross-realm bot, wouldn't it have a bot owner?

Reply:
As I mentioned here: https://github.com/zulip/zulip/pull/12372#discussion_r287963006, there's also a legacy data problem: Bots created before 2017 or so (when we added the `bot_owner` field) and not manually updated generally don't have an owner.